plugins/modules/foreman_interface.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2020 Bernhard Hopfenmüller (ATIX AG)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with This program.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: foreman_image
+short_description: Manage Host Interfaces using Foreman API
+description:
+  - Create and Delete Host Interfaces using Foreman API
+author:
+  - "Bernhard Hopfenmüller (@Fobhep) ATIX AG"
+options:
+  host:
+    description:
+      - |
```

Comment:
In the other modules, description is a List of plain strings. Why the multiline feature here?

Reply:
no specific reason - I copy-pasted the docs header from some other module (job template) :D will fix it

Reply:
fixed